src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,28 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import java.security.Key;
+
+public class LoginPage {
+
+    private WebDriver driver;
+    private By emailField = By.id("email");
+    private By passwordField = By.id("pass");
+    private By buttonSingin = By.id("send2");
+
+    public LoginPage(WebDriver driver){this.driver=driver;}
+
+    public  void Writeemail (String word){
+        driver.findElement(emailField).sendKeys(word, Keys.ENTER);
+    }
+
+    public void WritePassword (String word){
```

Comment:
the method cannot start with a capital letter, I suggest a more telling name, perhaps, `typePassword()`, the input value should be `password`